service/controller/key/key.go

```diff
@@ -86,29 +86,33 @@ func ClusterOrganization(clusterGuestConfig v1alpha1.ClusterGuestConfig) string
 func CommonAppSpecs() []AppSpec {
 	return []AppSpec{
 		{
-			App:             "cert-exporter",
+			// chart-operator must be installed first so the chart CRD is
+			// created in the tenant cluster.
+			App:             "chart-operator",
 			Catalog:         "default",
-			Chart:           "cert-exporter",
-			Namespace:       metav1.NamespaceSystem,
+			Chart:           "chart-operator",
+			Namespace:       "giantswarm",
 			UseUpgradeForce: true,
-			Version:         "1.2.1",
+			Version:         "0.11.2",
 		},
 		{
+			// coredns must be installed second as its a requirement for other
+			// apps.
```

Comment:
I think the next version of the app CRD is going to need to handle dependencies. Until then this ensures the apps are installed in a single loop as soon as the tenant cluster is up.